.github/workflows/job.module-info.yml

```
@@ -91,6 +94,26 @@ jobs:
+        # a script to determine the version of golangci-lint used in the project
+        if [[ -f "${{ inputs.path }}/.golangci.yml" ]]; then
+          lintVersion=$(cat "${{ inputs.path }}/.golangci.yml" | sed -n "s/^version: \"\([0-9]*\)\"$/\1/p")
+          if [[ -z "$lintVersion" ]]; then
+            echo "::error::no version found in .golangci.yml"
+            exit 1
+          fi
+        fi
+
+        # if there was no .golangci.yml file then assume golangci-lint v1.x
+        [[ -z $lintVersion ]] && lintVersion="1"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)